tests/Meziantou.Framework.ResxSourceGenerator.GeneratorTests/Resource1.es.resx

```diff
@@ -0,0 +1,27 @@
+<root>

```

Comment:
Can you remove this file as it is not needed for the RelativeDate?

Reply:
Sure, I will. It was created automatically, actually I didn't know its purpose 😆